app-emulation/vagrant/vagrant-2.2.14.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+USE_RUBY="ruby25 ruby26"
```

Comment:
Looks like all dependencies are in place for ruby27 as well?

Reply:
Yes I am adding ruby27 support.